### PR TITLE
Release v2.2.0

### PR DESCRIPTION
**Pull request by flowirtz**
## ToDos before merging
- [x] merge two new plugins into new embed style
- [x] remove `console.log` statement that jan doesn't like
- [x] Get an overview what's broken/too WIP/whatever and clean code a little for new maintainers 😊 
- [x] fix code block `onEnter` (see #100)
- [x] Finish & merge the new `<embed>` plugin started by Flo

### CODE REVIEW

**Comment by janrenz** [comment]
editing the title seems broken

**Comment by FWirtz** [comment]
> editing the title seems broken

@janrenz can't fully confirm.
Clicking on an empty title doesn't always work if you click on the placeholder text. If you click on it on the very far left of the row it works.
This is a known bug that I need to investigate.

For a given title editing should work alright.

Can you confirm?

**Comment by janrenz** [comment]
Confirmed

File: README.md [hunk 1]
```diff
@@ -20,12 +21,18 @@ yarn start
...
 The docs can be found [here](https://edtrio-docs.netlify.com/).
 
 ## Contribute
+
 We very much welcome any kinds of contributions! Have a look at our [issue tracker](https://github.com/schul-cloud/edtrio/issues) or [project board roadmap](https://github.com/schul-cloud/edtrio/projects/1) for inspiration.
+
 - Issue Tracker: https://github.com/schul-cloud/edtrio/issues
 - Source Code: https://github.com/schul-cloud/edtrio
 
+For some tooling explanations and guidelines, please head on over to the [docs](https://edtrio-docs.netlify.com/).
```
Comment (FWirtz) [inline README.md]: hm let's maybe deeplink to the actual "Guides/Development>Tooling" section no? What do you think @Atyanar? 😊 Can do this on Monday, will add a todo for me if that's ok for you.

**Pull request merged**